src/components/Global/LogStatusButton/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { Button } from '../index';
+import { signIn, signOut } from 'next-auth/react';
+import React from 'react';
+
+const LogStatusButton = ({ status }) => {
+  return status === 'authenticated' ? (
+    <Button style={'flat'} content={'Logout'} onClick={() => signOut()} />
+  ) : (
+    <Button style={'flat'} content={'Login'} onClick={() => signIn()} />
```

Comment:
I think we should put a TODO: redirect to /auth/login for all functions like this

Reply:
Like redirect instead of the `signIn()`/`signOut()`?